builtin/providers/aws/config.go

```diff
@@ -37,11 +39,17 @@ type AWSClient struct {
 // Client configures and returns a fully initailized AWSClient
 func (c *Config) Client() (interface{}, error) {
 	var client AWSClient
+	var errs []error
+
+	log.Println("[INFO] Detecting AWS creds via aws-sdk-go")
+	credsProvider, err := c.getCredentialsProvider()
+	if err != nil {
+		errs = append(errs, err)
+	}
 
 	// Get the auth and region. This can fail if keys/regions were not
 	// specified and we're attempting to use the environment.
-	var errs []error
-	log.Println("[INFO] Building AWS auth structure")
+	log.Println("[INFO] Building AWS auth structure via goamz")
 	auth, err := c.AWSAuth()
```

Comment:
Ah, good point, I was wondering what was the point of simulating like it all came from the TF config (in `AWSAuth `) :)

I will change it.